src/main/scala/amba/apb/ToTL.scala

```diff
@@ -45,23 +45,41 @@ class APBToTL()(implicit p: Parameters) extends LazyModule
       require (edgeOut.manager.minLatency >= 1)
       assert (!(in.psel && !in.penable && out.d.valid))
 
+
+      val beat = TransferSizes(1, beatBytes)
```

Comment:
You could as well just say TransferSizes(beatBytes, beatBytes), which might hypothetically give you a better QoR.

Reply:
Ah, `TransferSizes` is a range. got it.